TombEngine/Game/control/control.cpp

```diff
@@ -271,6 +271,13 @@ unsigned CALLBACK GameMain(void *)
 	else
 		g_Renderer.RenderTitleImage();
 
+	//Check if there is the title in Gameflow.lua
```

Comment:
Can you move this code somewhere higher in hierarchy? Lack of any levels in script is a very low-level error, so I suggest moving this to somewhere around `LoadFlowScript`. Proper way of throwing such error would be producing an exception in that function which will feature custom error message. Then TEN will automatically catch it, log it and exit around line 324 in winmain.cpp.